package CfgMagic;
import "CfgCommon.proto";



message Magic {
	required uint32 attrType = 1;
	required uint32 color = 2;
	required uint32 exp = 3;
	required uint32 level = 4;
	required uint32 lock = 5;
	required uint32 type = 6;
	required uint32 id = 7;
	required uint64 smelt = 8;
	repeated uint32 attrCal =9;
	repeated uint32 expCal =10;
	repeated uint64 smeltCal =11;
	repeated CfgCommon.Attribute attr = 12;
}

message Magics {
	repeated Magic magics = 1;
}


message MagicTower {
	required uint32 id = 1;
	required uint32 dailyReward  =2;
	repeated CfgCommon.Reward reward =3;

}

message MagicTowers {
	repeated MagicTower magic_towers = 1;
}
